package.json

```diff
@@ -14,15 +14,16 @@
   "main": "index.js",
   "scripts": {
     "test": "lab -cv",
-    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/stylish.js --exclude node_modules"
+    "lint": "jshint test index.js --reporter node_modules/jshint-stylish/index.js --exclude node_modules"
```

Comment:
@erikkemperman it's not that I'm not happy with the undertaker PR, it's just more complicated than I can look over in the short bursts of time I have :beers: